react-components/package.json

```diff
@@ -1,86 +1,85 @@
 {
-  "name": "react-components",
-  "version": "0.1.0",
-  "homepage": "/de/react_components",
-  "private": true,
-  "devDependencies": {
-    "@babel/plugin-proposal-class-properties": "^7.3.0",
-    "@babel/plugin-proposal-decorators": "^7.3.0",
-    "@storybook/addon-actions": "^5.0.10",
-    "@storybook/addon-console": "^1.1.0",
-    "@storybook/addon-knobs": "^5.0.10",
-    "@storybook/addon-links": "^5.0.10",
-    "@storybook/addons": "^5.0.10",
-    "@storybook/react": "^5.0.10",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "husky": "^1.3.1",
-    "jest-junit-reporter": "^1.1.0",
-    "prettier": "^1.16.4",
-    "prop-types": "^15.6.2"
-  },
-  "dependencies": {
-    "@cyverse-de/de-components": "^0.2.5",
-    "@material-ui/core": "^3.9.3",
-    "@material-ui/icons": "^3.0.2",
-    "add": "^2.0.6",
-    "classnames": "^2.2.6",
-    "formik": "^1.5.0",
-    "intro.js": "^2.9.3",
-    "moment": "^2.22.1",
-    "object-assign": "4.1.1",
-    "precise-commits": "^1.0.2",
-    "promise": "8.0.1",
-    "rc-tree": "^1.14.10",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-highlighter": "^0.4.2",
-    "react-intl": "^2.8.0",
-    "react-player": "^1.6.4",
-    "react-redux": "^5.0.7",
-    "react-scripts": "2.1.8",
-    "react-select": "^2.0.0",
-    "redux": "^4.0.0",
-    "redux-form": "^7.4.2",
-    "sanitize-html": "^1.20.0",
-    "showdown": "^1.9.0",
-    "sockette": "2.0.0",
-    "sw-precache-webpack-plugin": "0.11.5",
-    "whatwg-fetch": "2.0.3"
-  },
-  "scripts": {
-    "start": "npm run storybook",
-    "build": "REACT_APP_MODE=development node scripts/build.js",
-    "prod": "REACT_APP_MODE=production node scripts/build.js",
-    "test": "react-scripts test --env=jsdom",
-    "test:ci": "CI=true react-scripts test --env=jsdom --ci --testResultsProcessor ./node_modules/jest-junit-reporter",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
-    "prettier": "prettier --write 'src/**/*.js' 'stories/**/*.js'"
-  },
-  "babel": {
-    "presets": [
-      "react-app"
+    "name": "react-components",
+    "version": "0.1.0",
+    "homepage": "/de/react_components",
+    "private": true,
+    "devDependencies": {
+        "@babel/plugin-proposal-class-properties": "^7.3.0",
+        "@babel/plugin-proposal-decorators": "^7.3.0",
+        "@storybook/addon-actions": "^5.0.10",
+        "@storybook/addon-console": "^1.1.0",
+        "@storybook/addon-knobs": "^5.0.10",
+        "@storybook/addon-links": "^5.0.10",
+        "@storybook/addons": "^5.0.10",
+        "@storybook/react": "^5.0.10",
+        "babel-plugin-transform-class-properties": "^6.24.1",
+        "husky": "^1.3.1",
+        "jest-junit-reporter": "^1.1.0",
+        "prettier": "^1.16.4",
+        "pretty-quick": "^1.10.0",
+        "prop-types": "^15.6.2"
+    },
+    "dependencies": {
+        "@cyverse-de/ui-lib": "file:../../ui-lib",
```

Comment:
Will update this to pull from `npm` once https://github.com/cyverse-de/ui-lib/pull/7 is merged.